src/widgets/textlabel.h

```diff
@@ -2,7 +2,6 @@
 #define TEXTLABEL_H
 
 #include <QLabel>
-#include <QMouseEvent>
```

Comment:
QMouseEvent is used in the header for the mouseReleaseEvent override, on certain toolchains this is a build error.